draft-ietf-quic-recovery.md

```diff
@@ -1122,15 +1123,21 @@ Pseudocode for OnLossDetectionTimeout follows:
 ~~~
 OnLossDetectionTimeout():
   loss_time, pn_space = GetEarliestLossTime()
-  if (loss_time != 0):
-    // Time threshold loss Detection
-    DetectLostPackets(pn_space)
   // Retransmit crypto data if no packets were lost
   // and there are still crypto packets in flight.
-  else if (crypto packets are in flight):
+  if (crypto packets are in flight):
     // Crypto retransmission timeout.
     RetransmitUnackedCryptoData()
     crypto_count++
+  else if (endpoint is client without 1-RTT keys):
+    if (handshake keys available)
```

Comment:
```suggestion
    if (handshake keys available):
```

Unfortunately, this is still incorrect.  As we discussed on https://github.com/quicwg/base-drafts/issues/2519#issuecomment-472671420 there is a requirement to pad any time the client needs to send an Initial packet (it's a little more complicated in practice, but this is what the resolution boils down to).

Reply:
If the availability of Handshake keys isn't a strong enough condition for that, what should be used instead? I'd be inclined to just fall back on `SendOnePacket` and let the nontrivial padding logic be implied by the text elsewhere, but that's where we started for this fragment.

Reply:
If you need to send an Initial packet, pad.